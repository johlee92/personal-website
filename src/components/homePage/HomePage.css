.homepage-message {
    text-align: center;
}