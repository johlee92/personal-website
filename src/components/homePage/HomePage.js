import React, { Component } from 'react';
import './HomePage.css';

class HomePage extends Component {
    render() {
        return (
            <div className="homepage-message">
                Web Developer. Engineer. <br/>
                Ventures. Investing. Strategy. <br/>
                Adventures. Burner. Athlete. <br/>
                Denver. San Francisco. <br/>
            </div>
        )
    }
}

export default HomePage;