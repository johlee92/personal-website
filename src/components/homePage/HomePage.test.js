import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import HomePage from './HomePage';

describe(`HomePage component`, () => {
    it('renders without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(
                <HomePage />
            , div);
        ReactDOM.unmountComponentAtNode(div);
    });

    it('renders the component as expected', () => {
        const tree = renderer
            .create(
                <HomePage />
            )
            .toJSON();
        expect(tree).toMatchSnapshot();  
    });
})
