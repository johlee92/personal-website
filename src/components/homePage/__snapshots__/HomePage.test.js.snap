// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage component renders the component as expected 1`] = `
<div
  className="homepage-message"
>
  Web Developer. Engineer. 
  <br />
  Ventures. Investing. Strategy. 
  <br />
  Adventures. Burner. Athlete. 
  <br />
  Denver. San Francisco. 
  <br />
</div>
`;
